src/user-lists/entities/user-list-contributor-stats.entity.ts

```diff
@@ -4,7 +4,7 @@ import { ApiModelProperty } from "@nestjs/swagger/dist/decorators/api-model-prop
 @Entity({ name: "user_list_contributors" })
 export class DbUserListContributorStat {
   @ApiModelProperty({
-    description: "User list collaborator's login",
+    description: "User list connection's login",
```

Comment:
This is fine but this description doesn't really (and probably didn't to begin with 🙈) make sense.

```suggestion
    description: "User list contributor's login",
```
might make more sense